flake.nix

```diff
@@ -48,8 +48,70 @@
 
             doCheck = false;
           };
+          default = zkllvm-blueprint;
         };
 
+        testList = [
+        "blueprint_algebra_fields_plonk_exponentiation_test"
+        "blueprint_algebra_curves_plonk_unified_addition_test"
+        "blueprint_algebra_curves_plonk_variable_base_scalar_mul_test"
+        "blueprint_verifiers_kimchi_sponge_oracles_test"
+        "blueprint_hashes_plonk_poseidon_test"
+        "blueprint_algebra_curves_plonk_endo_scalar_test"
+        "blueprint_algebra_fields_plonk_range_check_test"
+        "blueprint_algebra_fields_plonk_logic_and_flag_test"
+        "blueprint_algebra_fields_plonk_logic_or_flag_test"
+        "blueprint_algebra_fields_plonk_interpolation_test"
+        "blueprint_algebra_fields_plonk_non_native_addition_test"
+        "blueprint_algebra_fields_plonk_non_native_subtraction_test"
+        #"blueprint_algebra_fields_plonk_non_native_multiplication_test"
+        "blueprint_algebra_fields_plonk_non_native_range_test"
+        "blueprint_algebra_fields_plonk_non_native_reduction_test"
+        "blueprint_algebra_fields_plonk_non_native_bit_decomposition_test"
+        "blueprint_algebra_fields_plonk_non_native_bit_composition_test"
+        "blueprint_algebra_fields_plonk_non_native_bit_shift_constant_test"
+        "blueprint_algebra_fields_plonk_non_native_logic_ops_test"
+        "blueprint_algebra_fields_plonk_non_native_lookup_logic_ops_test"
+        "blueprint_algebra_fields_plonk_non_native_comparison_checked_test"
+        "blueprint_algebra_fields_plonk_non_native_comparison_unchecked_test"
+        "blueprint_algebra_fields_plonk_non_native_comparison_flag_test"
+        "blueprint_algebra_fields_plonk_non_native_equality_flag_test"
+        "blueprint_algebra_fields_plonk_non_native_division_remainder_test"
+        #"blueprint_non_native_plonk_scalar_non_native_range_test"
+        "blueprint_non_native_plonk_bool_scalar_multiplication_test"
+        #"blueprint_non_native_plonk_add_mul_zkllvm_compatible_test"
+        "blueprint_hashes_plonk_decomposition_test"
+        "blueprint_verifiers_placeholder_fri_cosets_test"
+        "blueprint_hashes_plonk_sha256_process_test"
+        "blueprint_hashes_plonk_sha512_process_test"
+        "blueprint_hashes_plonk_sha256_test"
+        "blueprint_hashes_plonk_sha512_test"
+        "blueprint_algebra_fields_plonk_sqrt_test"
+        "blueprint_verifiers_placeholder_fri_lin_inter_test"
+        "blueprint_verifiers_placeholder_fri_array_swap_test"
+        #"blueprint_manifest_test"
+        #"blueprint_detail_huang_lu_test"
+        #"blueprint_private_input_test"
```

Comment:
These tests were originally enabled. Has something changed?

Reply:
Now the same tests as in master are enabled